docs/application/tizen-studio-0/setup/ubuntu.md

```diff
@@ -0,0 +1,197 @@
+   - Locate the **jdk-10** and the **javafx-sdk-11.0.2** directories.
+     > **Note**
+     >
+     >The **jdk-10**, **javafx-sdk-11.0.2** directories are created as a result of extraction. 
+   
+2. Copy all the files except the **src.zip** file from the **lib** sub directory located in the **javafx-sdk-11.0.2** directory.
+3. Paste all the copied files in the **lib** folder located in the **jdk-10** directory. 
+   >Note:
+   >
+   >- In this installation guide, setting up OpenJDK is covered. However, Oracle JDK versions until version 10 are also supported.
+   >- For Oracle JDK 10 installation steps, see the [Oracle official site](https://docs.oracle.com/javase/10/install/installation-jdk-and-jre-linux-platforms.htm#JSJIG-GUID-79FBE4A9-4254-461E-8EA7-A02D7979A161). 
+   >- If you have installed Oracle JDK 10 or below already, you can skip **Setting up OpenJDK**. 
+
+### Set Up Open JDK Path 
+
+To set up the JDK path, follow these steps:  
+
+1.	Open the terminal.
+2.	Type the following command in the terminal to set JAVA_HOME as the OpenJDK directory path, for example:
+	- `JAVA_HOME=/home/<username>/Desktop/openJDK/jdk-10.0.2/`.
+3.	Type the following commands in the terminal one after the other to complete the process:
+	- ` sudo apt update-alternatives --install /usr/bin/java java ${JAVA_HOME%*/}/bin/java 20000`
+	- ` sudo apt update-alternatives --install /usr/bin/javac javac ${JAVA_HOME%*/}/bin/javac 20000`
+4.	Type the following command to get a list of JDK versions installed:
+	- ` sudo apt update-alternatives --config java`
+	    > **Note**
+      >
+      > If there are multiple versions of JDK installed, enter the selection number to choose your desired JDK version.
+
+5. To configure the default Java compiler, type the following command:
+	
+   - ` sudo apt update-alternatives --config javac`
+	
+	   > **Note**
+     >
+     >**javac** is used for compilation. For more information on the differences between java and javac, see the [Oracle documentation](https://docs.oracle.com).
+	
+6.	Type the `java –version` command to verify whether the OpenJDK version 10 is installed.
+	
+## Install Dependencies
+
+The installer package consists of basic and immediate libraries. However, the Tizen Studio application requires additional libraries in order to work flawlessly. 
+
+To install the dependencies, open the terminal and execute the following commands: 
+
+- ` sudo apt install expect`
+- ` sudo apt install gtk2-engines-pixbuf`
+- ` sudo apt install libgnome2-0`
+- ` sudo apt install qemu-user-static`
+- ` sudo apt install libwebkitgtk-1.0-0`
+- ` sudo apt install libwebkitgtk-1.0-0 cpio rpm2cpio` 
+- ` sudo apt install gettext`
+- ` sudo apt install gksudo`
+- ` sudo apt install module-init-tools`
+- ` sudo apt install libudev-dev`
+- ` sudo apt install libsdl1.2debian`
+- ` sudo apt install bridge-utils`
+- ` sudo apt install openvpn`
```

Comment:
It is okay with one code block.

Reply:
I have made the necessary changes, please review and give your valuable comments.